### PR TITLE
drivers: watchdog: support for Nuvoton numaker WWDT

### PR DESCRIPTION
**Pull request by cyliangtw**
This PR is to add Nuvoton numaker window watchdog.
It's verified on board: NuMaker-PFM-M467.

### CODE REVIEW

File: drivers/watchdog/wdt_wwdt_numaker.c [hunk 1]
```diff
@@ -0,0 +1,301 @@
...
+	struct wwdt_numaker_data *data = dev->data;
+	const struct wwdt_numaker_config *cfg = dev->config;
+	WWDT_T *wwdt_base = cfg->wwdt_base;
+	uint32_t dbg_mask = 0U;
+
+	LOG_DBG("%s", __func__);
+	irq_disable(DT_INST_IRQN(0));
+
+	/* Validate watchdog already running */
+	if ((wwdt_base->CTL) & WWDT_CTL_WWDTEN_Msk) {
```
[the remark this answers is not in the document]
Comment (cyliangtw) [reply drivers/watchdog/wdt_wwdt_numaker.c]: Thanks and update to fulfill above 7 comments.

**Review by fabiobaltieri — DISMISSED** [review]

**Comment by cyliangtw** [comment]
Hi @ssekar15,
If you are available, please look into this PR.

**Comment by ssekar15** [comment]
> Hi @ssekar15, If you are available, please look into this PR.

bit hooked up, will have look shortly. Thanks

**Review by fabiobaltieri — APPROVED** [review]

**Review by ssekar15 — APPROVED** [review]
thanks

**Review by martinjaeger — APPROVED** [review]

**Comment by cyliangtw** [comment]
Hi @fabiobaltieri,
After assigned me [cyliangtw](https://github.com/cyliangtw), what should I do except waiting to merge ?

**Comment by fabiobaltieri** [comment]
@cyliangtw sorry unfortunately we missed the cut for RC1 so we have to wait until the 3.6 release, that's schedule for Feb 23.

**Comment by cyliangtw** [comment]
> @cyliangtw sorry unfortunately we missed the cut for RC1 so we have to wait until the 3.6 release, that's schedule for Feb 23.

@fabiobaltieri ,
never mind, end of Feb. is fine.

**Comment by cyliangtw** [comment]
Hi @aescolar,
This PR is not to change any file in \zephyr\boards\arm\numaker_pfm_m467\, or any file in \zephyr\soc\.
It seems not hwmv2-likely-conflict.

**Comment by aescolar** [comment]
@cyliangtw This label is only meant as an aid for the release engineering team, and only implies the PR will need to wait, hopefully, less than 1 week until it is merged.
Due to the work in the hwmv2 branch, and its imminent merge in the main branch, we are being overly cautious with merging PRs that change something which may conflict, specially samples/tests which change overlays or test definitions to avoid merge conflicts with that branch.

**Comment by cyliangtw** [comment]
@aescolar ,
Thanks, I see.

**Pull request merged**